manifests/w/Workday,Inc/PlanningforExcel/2022.105.2718.2929/Workday,Inc.PlanningforExcel.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Workday,Inc.PlanningforExcel
+PackageVersion: 2022.105.2718.2929
+Installers:
+- Architecture: neutral
+  InstallerType: burn
+  InstallerUrl: https://clickonce.adaptiveinsights.com/planning/latest/WorkdayAdaptivePlanningExcelSetup.exe
+  InstallerSha256: 5ED4DC5634F1E7F045E5301B480A83104BB829507D87DA15AC7FC19B91122B19

```

```suggestion
  InstallerSha256: 5ED4DC5634F1E7F045E5301B480A83104BB829507D87DA15AC7FC19B91122B19
	ProductCode: '{fbefb54e-0283-4ca5-a69a-e3ec6b1fa22b}'
```
Because this application has two ARP entries with the same DisplayName, the product code should be used to ensure the correct one is detected